src/components/Inputs/checkbox/CheckBox.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './CheckBox.scss';
+import "../../typography/Text.scss"
+
+interface ICheckboxProps {
+    isChecked?: boolean;
+    label?: string;
+    onChange?: (newState: boolean) => void;
```

Comment:
V checkbox komponento daš not property z imenom isChecked, onChange probaj obdržati isto ime. Da nekdo ko pride lahko hitro poveže kateri property se bo dejansko spremenil.

`onChange?: (isChecked: boolean) => void;
`